src/plugins/minimal_scene/MinimalScene.hh

```diff
@@ -205,23 +205,6 @@ namespace plugins
     /// \brief Flag to indicate texture size has changed.
     public: bool textureDirty = false;
 
-    /// \brief Scene service. If not empty, a request will be made to get the
-    /// scene information using this service and the renderer will populate the
-    /// scene based on the response data
-    public: std::string sceneService;
```

Comment:
Removing data members and functions doesn't break abi? Is this because it's a plugin?

Reply:
Yeah we're not installing this header file, so we don't need to worry about ABI